bedrock/products/templates/products/monitor/cms/index.html

```diff
@@ -62,7 +62,6 @@ <h2 class="mzp-u-title-md">{{ page.articles_heading }}</h2>
       {{ card(
         link_url=article.url,
         aspect_ratio="has-aspect-icon",
```

Comment:
@stephaniehobson I suppose this `has-aspect-icon` class doesn't make sense anymore since we're not showing an icon. Do you know what fits best here?